testgrid/config/config.yaml

```diff
@@ -3572,6 +3576,10 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-cni-flannel
     alert_options:
       alert_mail_to_addresses: 'kubernetes-sig-network-test-failures@googlegroups.com'
+  - name: kubeadm-gce-dns-coredns
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-dns-coredns
+    alert_options:
+      alert_mail_to_addresses: 'kubernetes-sig-network-test-failures@googlegroups.com'
```

Comment:
To me, this alert configuration contradicts your other setting:

    +    "sigOwners": [
    +      "sig-cluster-lifecycle"
    +    ]

Should the owning SIG also be networking?

Reply:
hum.. here i am copying the job 'kubeadm-gce', which has those entries (along with other addons like calico, flannel, etc...). I am not sure where it should go.

@luxas: do you an idea who should be the owner of that job ?

Reply:
I would go with this SIG for now.
If that needs to change then it means a bunch of jobs will have to be changed.

Reply:
I'm fine with this as-is